client/modules/datafiles/src/DatafilesToolbar/DatafilesToolbar.tsx

```diff
@@ -103,7 +104,7 @@ export const DatafilesToolbar: React.FC<{ searchInput?: React.ReactNode }> = ({
         // Disable downloads from frontera.work until we have a non-flaky mount on ds-download.
         canDownload:
           selectedFiles.length >= 1 &&
-          system !== 'designsafe.storage.frontera.work' &&
+          system !== USER_WORK_SYSTEM &&
```

Comment:
I didn't realize this was set - great, so user the Work tapis download feature can be done later and no current functionality is lost 👍